packages/buttons/src/elements/ButtonGroup.tsx

```diff
@@ -14,6 +14,11 @@ import { ButtonGroupContext } from '../utils/useButtonGroupContext';
 
 export interface IButtonGroupProps extends HTMLAttributes<HTMLDivElement> {
   selectedItem?: any;
```

Comment:
Please document `selectedItem`. You can see from https://garden.zendesk.com/components/button-group#buttongroup that it is missing.

Reply:
Sorry I missed that one. Fixed in latest push.